go/common/sgx/ias/avr.go

```diff
@@ -231,6 +237,7 @@ func (a *AttestationVerificationReport) validate() error { // nolint: gocyclo
 	}
 
 	// TODO: Enforce version once version 3 test vectors are available.
+	// Enforce to version 4 now?
```

Comment:
Should we enforce version here now?

Reply:
I think we should enforce version 4, yeah.

cc @Yawning about the test vectors.

Reply:
i didn't find anything, so i just updated the comment for now :grin: